[2025-02-21T07:30:33.919+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T02:08:00+00:00 [queued]>
[2025-02-21T07:30:33.973+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T02:08:00+00:00 [queued]>
[2025-02-21T07:30:33.975+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-02-21T07:30:34.198+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_task> on 2025-01-01 02:08:00+00:00
[2025-02-21T07:30:34.241+0000] {standard_task_runner.py:60} INFO - Started process 1820 to run task
[2025-02-21T07:30:34.372+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'optum_etl_dag1', 'load_task', 'scheduled__2025-01-01T02:08:00+00:00', '--job-id', '291', '--raw', '--subdir', 'DAGS_FOLDER/optum_etl_dag1.py', '--cfg-path', '/tmp/tmpx09n35r0']
[2025-02-21T07:30:34.454+0000] {standard_task_runner.py:88} INFO - Job 291: Subtask load_task
[2025-02-21T07:30:34.793+0000] {task_command.py:423} INFO - Running <TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T02:08:00+00:00 [running]> on host 491940227afd
[2025-02-21T07:30:35.388+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='optum_etl_dag1' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T02:08:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-01T02:08:00+00:00'
[2025-02-21T07:30:35.563+0000] {logging_mixin.py:188} INFO - Loaded data: {'name': 'AIRFLOW', 'type': 'ORCHESTRATION'}
[2025-02-21T07:30:35.579+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-02-21T07:30:35.661+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=optum_etl_dag1, task_id=load_task, execution_date=20250101T020800, start_date=20250221T073033, end_date=20250221T073035
[2025-02-21T07:30:35.864+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-21T07:30:35.938+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
